dist/migration/class-notice.php

```diff
@@ -61,7 +61,7 @@ class Notice {
 	 *
 	 * @var string
 	 */
-	const NOTICE_CAPABILITY = 'install_plugins';
+	const CAPABILITY = 'install_plugins';
```

Comment:
Now that this is referenced in another class, `Notice::CAPABILITY` seems better than `Notice::NOTICE_CAPABILITY`

Reply:
That's a good idea, but I don't see another class where it might fit better.

Maybe [class-install-genesis-blocks-api.php](https://github.com/studiopress/atomic-blocks/blob/3e111746cebc5cc6b79b7f9b6ab9ae9ba24f411e/dist/migration/class-install-genesis-blocks-api.php), but there doesn't look to be a general `Migration` class.

Maybe a global, but I'm not sure if that'd be an improvement.